GitHubLabels.ps1

```diff
@@ -85,9 +108,33 @@ function Get-GitHubLabel
         'RepositoryName' = (Get-PiiSafeString -PlainText $RepositoryName)
     }
 
+    if ($PSBoundParameters.ContainsKey('Issue'))
+    {
+        $uriFragment = "/repos/$OwnerName/$RepositoryName/issues/$Issue/labels"
+        $description = "Getting labels for Issue $Issue in $RepositoryName"
+    }
+    elseif ($PSBoundParameters.ContainsKey('Milestone'))
+    {
+        $uriFragment = "/repos/$OwnerName/$RepositoryName/milestones/$Milestone/labels"
+        $description = "Getting labels for Milestone $Milestone in $RepositoryName"
+    }
+    else
+    {
+        $uriFragment = "repos/$OwnerName/$RepositoryName/labels/$Name"
+
+        if ($PSBoundParameters.ContainsKey('Name'))
+        {
+            $description =  "Getting label $Name for $RepositoryName"
+        }
+        else
+        {
+            $description = "Getting labels for $RepositoryName"
+        }
+    }
+
     $params = @{
-        'UriFragment' = "repos/$OwnerName/$RepositoryName/labels/$Name"
-        'Description' =  "Getting all labels for $RepositoryName"
+        'UriFragment' = $uriFragment
```

Comment:
I mentioned this in @etgottli's PR as well.  I'm surprised that PSScriptAnalyzer isn't catching this.  `$uriFragment` and `$description` aren't initially defined in the same scope as this.  While PowerShell supports this in theory, it's safer to explicitly define them in the same scope (outside/before your `if/then/else`.